src/lib/evil/unposix/sys/types.h

```diff
@@ -1,21 +1,32 @@
 #ifndef UNPOSIX_SYS_TYPES_H
 #define UNPOSIX_SYS_TYPES_H
 
-#ifdef _MSC_VER
-#ifndef WIN32_LEAN_AND_MEAN
-# define WIN32_LEAN_AND_MEAN
+#ifndef _CRT_DECLARE_NONSTDC_NAMES
```

Comment:
What does it do?

Reply:
https://github.com/expertisesolutions/efl/pull/9#discussion_r413389343